lib/pf/cmd/pf/generatemariadbconfig.pm

```diff
@@ -101,6 +101,8 @@ sub _run {
         server_ip => $management_network ? ($management_network->{Tvip} // $management_network->{Tip}) : "",
         performance_schema => _pf_config_val("database_advanced", "performance_schema"),
         max_connect_errors => _pf_config_val("database_advanced", "max_connect_errors"),
+        net_read_timeout => normalize_time(_pf_config_val("database_advanced", "net_read_timeout")),
```

Comment:
No need to call normalize time.
```suggestion
        net_read_timeout => _pf_config_val("database_advanced", "net_read_timeout"),
```

Reply:
if i am not wrong _pf_config_val return the raw value, so i need to convert it.

Reply:
This was already done in pfconfig no need to do it here.

Reply:
```

Reply:
If that is the case the we are good.